aqua/yabai/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                yabai
+
+github.setup        koekeishiya yabai 3.3.0 v
+revision            0
+
+categories          aqua
+platforms           darwin
+license             MIT
+maintainers         openmaintainer
+
+description         Tiling window management for the Mac
+
+long_description    yabai is a window management utility that is designed \
+                    to work as an extension to the built-in window manager \
+                    of macOS
+
+homepage            https://github.com/koekeishiya/yabai
```

Comment:
This line is redundant and should be removed. It is set to this value by the `github.setup` command above.

Reply:
Fixed